### PR TITLE
[16.0][MIG] account_move_line_stock_info: Migration to 16.0

### PR DESCRIPTION
**Pull request by BT-atornero**

### CODE REVIEW

File: account_move_line_stock_info/tests/test_account_move_line_stock_move.py [hunk 1]
```diff
@@ -121,18 +114,16 @@ def _create_picking(self, picking_type, location, location_dest):
                 "picking_type_id": picking_type.id,
                 "location_id": location.id,
                 "location_dest_id": location_dest.id,
-                "move_lines": [
+                "move_line_ids": [
```
Comment (rousseldenis) [inline account_move_line_stock_info/tests/test_account_move_line_stock_move.py]: That is not the same.

Change to 'move_ids' instead

Comment (BT-atornero) [reply account_move_line_stock_info/tests/test_account_move_line_stock_move.py]: do you mean 'move_lines'? because 'move_ids' does not seem to exist

Comment (rousseldenis) [reply account_move_line_stock_info/tests/test_account_move_line_stock_move.py]: Of course it exists : https://github.com/odoo/odoo/blob/16.0/addons/stock/models/stock_picking.py#L346

Comment (rousseldenis) [reply account_move_line_stock_info/tests/test_account_move_line_stock_move.py]: move_lines has been renamed to move_ids : https://github.com/odoo/odoo/commit/3a1473c75c5fabfa9b060c77eed729f74f14b559

**Comment by BT-atornero** [comment]
there is another PR with the migration of this module https://github.com/OCA/stock-logistics-warehouse/pull/1645

**Pull request closed without merging**